app/templates/events/view/scheduler.hbs

```diff
@@ -15,7 +15,7 @@
         {{/each}}
       </div>
     </div>
-    <div class="thirteen wide column">
+    <div class="{{if device.isMobile 'ui segment' 'thirteen wide column'}}">
```

Comment:
Does it need to be a segment on mobile ? Can you show me a comparison of how it looks ?

Reply:
@niranjan94 Without the segmented view :- 
<img width="830" alt="Screenshot 2019-06-03 at 5 06 23 AM" src="https://user-images.githubusercontent.com/44091822/58768746-6e753900-85bd-11e9-90f1-bb77703a9c21.png">

With Segmented View :- 
<img width="784" alt="Screenshot 2019-06-03 at 5 06 42 AM" src="https://user-images.githubusercontent.com/44091822/58768747-7634dd80-85bd-11e9-98b6-4e41c2b0bf11.png">

Reply:
I think the segment can be removed. Its not needed here as the mobile device width is already too low to be taken up by it. Just ensure there is some space above the date/period selection buttons

Reply:
Oh will add a `Hidden Divider `